app/views/admin_request/show.html.erb

```diff
@@ -312,6 +312,18 @@
                 </td>
               </tr>
             <% end %>
+            <tr>
+              <td class="span3">
+                <b>Raw body</b>
+              <td>
+                <% truncated_value = truncate(h(outgoing_message.raw_body), length: 400, omission: '') { link_to '…', '#', title: 'Toggle hidden', class: 'toggle-hidden' } %>
+                    <%= simple_format(truncated_value) %>
+                    <div style="display:none;"><%= simple_format(outgoing_message.raw_body) %></div>
+                <div style="display:none;">
+                  <%= simple_format(outgoing_message.raw_body) %>
+                </div>
```

Comment:
This includes the full text twice, so clicking `toggle-hidden` results in two copies:

![screen shot 2018-10-24 at 13 04 27](https://user-images.githubusercontent.com/282788/47429170-66cef580-d78d-11e8-9c09-a23881de9fe7.png)

The indenting here was really confusing at first glance.

I _think_ the intent was for the final code to be:

```erb
<% truncated_value = truncate(h(outgoing_message.raw_body), length: 400, omission: '') { link_to '…', '#', title: 'Toggle hidden', class: 'toggle-hidden' } %>
<%= simple_format(truncated_value) %>
<div style="display:none;">
  <%= simple_format(outgoing_message.raw_body) %>
</div>
```